cvat/apps/dataset_manager/formats/cvat.py

```diff
@@ -441,7 +441,7 @@ def load(file_object, annotations):
             elif el.tag == 'image':
                 image_is_opened = True
                 frame_id = annotations.abs_frame_id(match_dm_item(
-                    DatasetItem(id=el.attrib['name'],
+                    DatasetItem(id=osp.splitext(el.attrib['name'])[0],
```

Comment:
This change will break support for images with same filename and different extension in CVAT format, which is the only format that supports it.